src/main/java/com/treasure/hunt/game/GameEngine.java

```diff
@@ -27,7 +27,13 @@
  */
 @Requires(hider = Hider.class, searcher = Searcher.class)
 public class GameEngine {
+    /**
+     * The height of the playing area.
+     */
     public static final int HEIGHT = 200;
+    /**
+     * The width of the playing area.
+     */
     public static final int WIDTH = 200;
```

Comment:
We really need to make that settable, I don't think ewe wanna have such a restriction at all

Reply:
But don't worry this is out of the scope of this PR

Reply:
This is an example for issue #103